### ISSUE TITLE
Specifying format on properties?

### ISSUE THREAD

**Issue by ChlorideCull** [microsoft/OpenAPI.NET.CSharpAnnotations#220]
I can't find any way to set a format such as "date" on a property, as [defined in the OAS 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#data-types) spec. 

It's automatically set to "date-time" on DateTime objects, but I want to set it to "date" as that's the format that the JsonConverter I have on the property converts it to.

**Comment by Shwetap05** [comment]
@ChlorideCull Yes we currently do not have a way to specify "date" this needs to be fixed.  I will get back to you on the ETA for the fix.

**Comment by christopher-bonitz** [comment]
@Shwetap05 did you have a chance to get an ETA on this?
I've stumbled upon a similar case where I use a string field to circumvent the issue, had some other issues longer down the pipe, so would prefer to just mark date as a format.
